package.json

```diff
@@ -27,6 +27,7 @@
     "file-loader": "0.11.2",
     "fork-ts-checker-webpack-plugin": "^0.2.8",
     "fs-extra": "3.0.1",
+    "history": "^4.7.2",
```

Comment:
Do we really need new package? 
React Router is mostly a wrapper around the history library. history handles interaction with the browser's window.history for you with its browser and hash histories.
https://stackoverflow.com/questions/31079081/programmatically-navigate-using-react-router